packages/dids/readme.md

```diff
@@ -1,9 +1,9 @@
 
 # DID Manager
 
-DID manager for the dock wallet sdk
+DID stands for Decentralized IDentifiers. DIDs are meant to be globally unique identifiers that allow their owner to prove cryptographic control over them. A DID identifies any subject (e.g., a person, organization, thing, data model, abstract entity, etc.) that the controller of the DID decides that it identifies.
```

Comment:
should we call out which DID methods we support here somewhere?

Reply:
I think it supports just DID key, right @osinakayah?

Reply:
Per the thread in Slack let's add `did:dock` as well.